Sources/Build/BuildDelegate.swift

```diff
@@ -235,6 +242,14 @@ public final class BuildDelegate: BuildSystemDelegate, SwiftCompilerOutputParser
     }
 
     public func commandStatusChanged(_ command: SPMLLBuild.Command, kind: CommandStatusKind) {
+        guard !isVerbose else { return }
+        guard command.shouldShowStatus else { return }
+        guard !swiftParsers.keys.contains(command.name) else { return }
```

Comment:
Is it safe to check this one without thread-safety?

Reply:
Should be okay as `swiftParsers` is never mutated. I should make it `let`.